getProxyAgentOptions error

- MSSQL Extension Version: 1.20.0
- VSCode Version: 1.80.1
- OS Version: Win11

Steps to Reproduce:

1. Upgrade mssql vscode module version from v 1.19.1 to 1.20.0
2. Try to connect to a SQL server

The following getProxyAgentOptions error is found in the output window

Downgrading to 1.19.1 fixes the problem.

I do have a proxy defined on my workstation (via environmental variables, Internet Connection configuration and settings.json)

> Initializing SQL tools service for the mssql extension.
> Note: mssql commands will be available after installing the service.
> 
> Platform: win32, x86_64 (Windows)
> 
> Installing SQL tools service to c:\Users\EVCallahan\.vscode\extensions\ms-mssql.mssql-1.20.0\sqltoolsservice\4.8.1.2\Windows.
> Downloading https://github.com/Microsoft/sqltoolsservice/releases/download/4.8.1.2/microsoft.sqltools.servicelayer-win-x64-net7.0.zip
> [ERROR] TypeError: Cannot read properties of undefined (reading 'getProxyAgentOptions')

--- a/src/languageservice/proxy.ts
+++ b/src/languageservice/proxy.ts
@@ -30,7 +30,7 @@ export function getProxyAgent(requestURL: Url, proxy?: string, strictSSL?: boole
 		return undefined;
 	}
 	const proxyEndpoint = parseUrl(proxyURL);
-	const opts = this.getProxyAgentOptions(requestURL, proxy, strictSSL);
+	const opts = getProxyAgentOptions(requestURL, proxy, strictSSL);
 	return proxyEndpoint.protocol === 'https:' ? new HttpsProxyAgent(opts as HttpsProxyAgentOptions) : new HttpProxyAgent(opts as HttpProxyAgentOptions);
 }
 

